lib/TrackerManager.py

```diff
@@ -0,0 +1,230 @@
+import os
+import sys
+import json
+from datetime import datetime
+from .SoftwareHttp import *
+from .blake2 import BLAKE2b
+# Add vendor directory to module search path
+# This needs to be here to load the snowplow_tracker library
+vendor_dir = os.path.abspath(os.path.join(os.path.dirname( __file__ ), '..', 'vendor'))
```

Comment:
Does this always work? Maybe something like the Sublime `find_resources(pattern)` might work?

Reply:
yeah seems to always work